src/core/DropdownCard/Dropdown/Dropdown.tsx

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import { Menu } from '@material-ui/core';
+import { useStyles } from './styles';
+import { BasePropsMenu } from '../base';
+
+interface CustomDropdownProps extends BasePropsMenu {
+  open: boolean;
+}
+const Dropdown: React.FC<CustomDropdownProps> = ({ children, open }) => {
+  const classes = useStyles();
+  return (
+    <div className={classes.dropdownDiv}>
+      <div className={classes.triangleUp}></div>
```

Comment:
can we merge this div , class with the above classes `dropdownDiv`,
and insted `classes.dropdownDiv`  can we have `classes.root`?